### DISCUSSION TITLE
How to use multiple selection on /buffer or /file？

### DISCUSSION THREAD

**Discussion by amojury** [olimorris/codecompanion.nvim#430] [Q&A] [answered]
I saw the pr. but don't know how to config it. Thanks

https://github.com/olimorris/codecompanion.nvim/pull/406

**Comment by olimorris** [comment, marked as answer]
You need to select Telescope or Mini.Pick. Then you use tab to select your buffers in the modal.

**Reply by fredrikaverpil** [reply]
Also works with fzf_lua 😄
